app/events/decidim/proposals/proposal_published_event.rb

```diff
@@ -0,0 +1,10 @@
+# app/events/decidim/proposals/proposal_published_event.rb
+module Decidim
+  module Proposals
+    class ProposalPublishedEvent < Decidim::Events::SimpleEvent
```

Comment:
```suggestion
    class AuthorConfirmationProposalEvent < Decidim::Events::SimpleEvent
```

To be easier to read, we should rename the class Event as follow and also rename the file to `app/events/decidim/proposals/author_confirmation_proposal_event.rb`